workspace/src/test/kotlin/com/cosmotech/workspace/azure/WorkspaceServiceImplTests.kt

```diff
@@ -271,6 +265,8 @@ class WorkspaceServiceImplTests {
     confirmVerified(cosmosTemplate)
   }
 
+  /*
+  Test cannot be perform for now since update checks now for JWT security context
```

Comment:
Why is this test commented ? It should be possible to adjust it (and eventually test the new behaviors) by mocking module-wide Kotlin functions added in `SecurityUtils.kt`.
See some examples in `ScenarioServiceImplTests`, where `Securityutils::getCurrentAuthentication` is [mocked before each test](https://github.com/Cosmo-Tech/cosmotech-api/blob/main/scenario/src/test/kotlin/com/cosmotech/scenario/azure/ScenarioServiceImplTests.kt#L105-L108) and [unmocked after each test](https://github.com/Cosmo-Tech/cosmotech-api/blob/main/scenario/src/test/kotlin/com/cosmotech/scenario/azure/ScenarioServiceImplTests.kt#L113-L116)

Reply:
todo

Reply:
done